2022-06-09 10:32:51,029 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2022-06-09 10:32:51,030 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from F:\django_\instagramProject\venv\lib\site-packages\instabot\bot\bot.py
2022-06-09 10:32:52,321 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2022-06-09 10:32:52,864 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [200]>
2022-06-09 10:32:53,357 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: qe/sync/ returned response: <Response [200]>
2022-06-09 10:32:53,844 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: launcher/sync/ returned response: <Response [200]>
2022-06-09 10:32:54,319 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/contact_point_prefill/ returned response: <Response [200]>
2022-06-09 10:33:02,035 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/login/ returned response: <Response [400]>
2022-06-09 10:33:02,036 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"challenge_required","challenge":{"url":"https://i.instagram.com/challenge/AXGUsXe_32xcyjZP_3DOSZWyQvtq9vedhCPNJAHW6JSEK5bSW7XYkZIVTOArPzitdWVR-zw/IiVkRB1gJcwnPLXz/","api_path":"/challenge/AXGUsXe_32xcyjZP_3DOSZWyQvtq9vedhCPNJAHW6JSEK5bSW7XYkZIVTOArPzitdWVR-zw/IiVkRB1gJcwnPLXz/","hide_webview_header":true,"lock":true,"logout":false,"native_flow":true,"flow_render_type":0},"status":"fail","error_type":"checkpoint_challenge_required"}'
2022-06-09 10:33:02,036 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2022-06-09 10:33:02,037 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: challenge_required
2022-06-09 10:33:02,038 - instabot version: 0.117.0 (api) - INFO - Error type: checkpoint_challenge_required
